test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/initialgainorloss/InitialGainOrLossControllerSpec.scala

```diff
@@ -139,6 +139,8 @@ class InitialGainOrLossControllerSpec
             ), { doc =>
               doc.select(".govuk-caption-xl").html()  should include("Initial gain or loss")
               doc.select("#initialGainOrLoss").html() should include("Did you make an initial gain or loss?")
+              doc.select("#initialGainOrLoss").html() should include("Initial gain amount")
+              doc.select("#initialGainOrLoss").html() should include("Initial loss amount")
```

Comment:
this is the same thing twice. Also, is it possible to have a stronger test which has `shouldBe` instead of `should include`? if you do `doc.select("...").text()` would that work?

Reply:
Ok, one was for loss and the other one was for gain. but sure.

Reply:
oh ok, I see that now. What would make it easier to test then is using a more specific selector or perhaps giving the different elements different id's to be able to assert that a precise element has a particular text

Reply:
Yes sure, I'll do that.